src/perform_step/predcorr.jl

```diff
@@ -0,0 +1,70 @@
+@muladd function perform_step!(integrator,cache::PCEulerConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u,W,p,f,g = integrator
+  @unpack theta,eta,bbprime = integrator.alg
+  dW = W.dW
+  if !is_diagonal_noise(integrator.sol.prob) || typeof(W.dW) <: Number
+    gdW_n = g(uprev,p,t)*dW
+  else
+    gdW_n = g(uprev,p,t).*dW
+  end
+
+  f_n = f(uprev,p,t)
+  ubar = @muladd uprev + dt*f_n + gdW_n
```

Comment:
`@muladd` on the function makes everything inside `@muladd`, so these aren't needed (there are probably some other functions in this library that need cleaning for that)

Reply:
Awesome - I did note that adding this did not change the speed. Will remove them.